src/stp/plays/PenaltyUsPrepare.cpp

```diff
@@ -17,19 +17,24 @@ PenaltyUsPrepare::PenaltyUsPrepare() : Play() {
     keepPlayInvariants.emplace_back(std::make_unique<invariant::PenaltyUsPrepareGameStateInvariant>());
 
     roles = std::array<std::unique_ptr<Role>, rtt::ai::Constants::ROBOT_COUNT()>{
-        std::make_unique<role::Formation>(role::Formation("keeper")),      std::make_unique<role::Formation>(role::Formation("kicker_formation")),
-        std::make_unique<role::Formation>(role::Formation("formation_0")), std::make_unique<role::Formation>(role::Formation("formation_1")),
-        std::make_unique<role::Formation>(role::Formation("formation_2")), std::make_unique<role::Formation>(role::Formation("formation_3")),
-        std::make_unique<role::Formation>(role::Formation("formation_4")), std::make_unique<role::Formation>(role::Formation("formation_5")),
-        std::make_unique<role::Formation>(role::Formation("formation_6")), std::make_unique<role::Formation>(role::Formation("formation_7")),
+        std::make_unique<role::Formation>(role::Formation("keeper")),
+        std::make_unique<role::Formation>(role::Formation("kicker_formation")),
+        std::make_unique<role::Formation>(role::Formation("formation_0")),
+        std::make_unique<role::Formation>(role::Formation("formation_1")),
+        std::make_unique<role::Formation>(role::Formation("formation_2")),
+        std::make_unique<role::Formation>(role::Formation("formation_3")),
+        std::make_unique<role::Formation>(role::Formation("formation_4")),
+        std::make_unique<role::Formation>(role::Formation("formation_5")),
+        std::make_unique<role::Formation>(role::Formation("formation_6")),
+        std::make_unique<role::Formation>(role::Formation("formation_7")),
         std::make_unique<role::Formation>(role::Formation("formation_8"))};
 }
 
 uint8_t PenaltyUsPrepare::score(world::World* world) noexcept { return 100; }
 
 void PenaltyUsPrepare::calculateInfoForRoles() noexcept {
-    auto width = field.getFieldWidth();
-    auto length = field.getFieldLength();
+    const auto xPosition = -4 * control_constants::ROBOT_RADIUS;
+    const auto yPosition = Constants::STD_TIMEOUT_TO_TOP() ? field.getFieldWidth() / 2.2 : -field.getFieldWidth() / 2.2;
```

Comment:
Same story as above; dont repeat yourself, and don't use magic values (e.g. 2.2).

Reply:
you can remember this as; you want to be DRY (Dont Repeat Yourself), and not WET (Write Everything Twice)